framework/src/actions/SetupDebugAction.C

```diff
@@ -83,6 +83,8 @@ SetupDebugAction::createOutputAction(const std::string & type, const std::string
   // Set the object parameters
   InputParameters & object_params = action->getObjectParams();
   object_params.set<bool>("_built_by_moose") = true;
+  if (type == "VariableResidualNormsDebugOutput")
```

Comment:
This looks like a hack. Do we know why this quit working? Does it have to do with the new `ExecFlagEnum` refactor? I'd prefer to not see the SetupDebugAction have to call out some other type by name.